src/blueapi/service/interface.py

```diff
@@ -48,10 +48,13 @@ def set_config(new_config: ApplicationConfig):
 @cache
 def context() -> BlueskyContext:
     ctx = BlueskyContext()
-    ctx.with_config(config().env)
     return ctx
 
 
+def instantiate_devices_in_context() -> None:
+    context().with_config(config().env)
```

Comment:
Is instantiating all of the devices all that with_config does on Context? If so it could use a better name, if not this function should get a better name

Reply:
Also no, thats not all it does. It registers the plans and devices then instantiates and connects the devices